src/main/java/core/basesyntax/model/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.model;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBERS_OF_FIGURES = 5;
+    private Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Figure figureRandom = null;
+        String colorRandom = supplier.getRandomColor().toLowerCase();
+
+        switch (random.nextInt(NUMBERS_OF_FIGURES)) {
+            case 0: // Square
+                double side = random.nextDouble();
+                figureRandom = new Square(colorRandom, side);
+                break;

```

Comment:
```suggestion
            case 0: // Square
                return getRandomSquare();
```
move creation to separate private methods - to make code easier to read